exchange/exchangetest/passthrough.json

```diff
@@ -0,0 +1,116 @@
+{
```

Comment:
Nitpick: can we rename these test files to something like:
`passthrough.json` --> `passthrough_root_and_imp.json`
`passthrough_multi_imp.json` --> `passthrough_imp_only.json`
`passthrough_only_request.json` --> `passthrough_root_only.json`